tests/e2e/specs/infobox.spec.js

```diff
@@ -53,7 +53,7 @@ describe('The infobox', () => {
                     cy.get('[data-cy="popover"]').should('be.visible')
                     cy.get('[data-cy="tooltip"]').should('not.exist')
 
-                    cy.get('[data-cy="toggle-floating-off"]').click()
+                    cy.get('[data-cy="toggle-floating-off"]').click({ force: true })
```

Comment:
not sure if using `force` is a good idea, that means that your HTML element is certainly behind something while trying to reach it (so there is some design/CSS issue anyway), no?

Reply:
The problem is that we would need to pan the map to see the controls. I added a media query so restrict the height on small viewports. This way we don't need to force the click.